tools/superdsm/superdsm.xml

```diff
@@ -14,19 +14,71 @@
+       --c2f_region_analysis_max_atom_norm_energy '${config.c2f_region_analysis_max_atom_norm_energy}'
+   #end if
+   #if str($config.c2f_region_analysis_max_cluster_marker_irregularity) != '':
+       --c2f_region_analysis_max_cluster_marker_irregularity '${config.c2f_region_analysis_max_cluster_marker_irregularity}'
+   #end if
+   #if str($config.dsm_alpha) != '':
+       --dsm_alpha '${config.dsm_alpha}'
+   #end if
+   #if str($config.dsm_AF_alpha) != '':
+       --dsm_AF_alpha '${config.dsm_AF_alpha}'
+   #end if
+   #if str($config.global_energy_minimization_beta) != '':
+       --global_energy_minimization_beta '${config.global_energy_minimization_beta}'
+   #end if
+   #if str($config.global_energy_minimization_AF_beta) != '':
+       --global_energy_minimization_AF_beta '${config.global_energy_minimization_AF_beta}'
+   #end if
+   #if str($config.postprocess_mask_max_distance) != '':
+       --postprocess_mask_max_distance '${config.postprocess_mask_max_distance}'
+   #end if
+   #if str($config.postprocess_mask_stdamp) != '':
+       --postprocess_mask_stdamp '${config.postprocess_mask_stdamp}'
+   #end if
+   #if str($config.postprocess_max_norm_energy) != '':
+       --postprocess_max_norm_energy '${config.postprocess_max_norm_energy}'
+   #end if
+   #if str($config.postprocess_min_contrast) != '':
+       --postprocess_min_contrast '${config.postprocess_min_contrast}'
+   #end if
+   #if str($config.postprocess_min_object_radius) != '':
+       --postprocess_min_object_radius '${config.postprocess_min_object_radius}'
+   #end if
    ]]>
    </command>
    <inputs>
         <param name="dataset" type="data" format="tiff,png" label="Dataset" />
         <param name="seg_border" type="integer" min="1" value="8" label="Width of the outlines (in pixels) of the segmentation results (overlays)" />
-        <!--
         <section name="config" title="Hyperparameters" expanded="false">
+            <param name="AF_scale" optional="true" type="float" value="" min="0" label="scale σ" help="The scale of the objects to be segmented. Leave empty to use the automatically determined value." />
+            <param name="c2f_region_analysis_min_atom_radius" optional="true" type="float" value="" min="0" label="min_atom_radius" help="No region determined by the Coarse-to-fine region analysis scheme is smaller than a circle of this radius (in terms of the surface area). Leave empty to use the automatically determined value." />
+            <param name="c2f_region_analysis_min_norm_energy_improvement" type="float" value="0.1" min="0" label="min_norm_energy_improvement" help="Each split performed during the computation of the atomic image regions must improve the normalized energy 𝑟(𝜔) of an image region 𝜔 by at least this factor. Given that an image region is split into the sub-regions 𝜔₁, 𝜔₂, the improvement of the split is defined by the fraction max{𝑟(𝜔)₁, 𝑟(𝜔₂)} / 𝑟(𝜔₁ ∪ 𝜔₂). Lower values of the fraction correspond to better improvements." />

```

Comment:
You could use argument="--c2f_region_analysis_min_norm_energy_improvement" here and everywhere else.
https://docs.galaxyproject.org/en/latest/dev/schema.html#id37

Reply:
@bgruening but what is the advantage of using `argument` instead of `name`? The only difference that I see from the docs is that the argument is rendered in the help section (which I think would be rather confusing for the user in this case).

Reply:
It gives you nice integration with the help="" and its "better" annotated.
There are ideas and projects floating around that are parsing the tool annotation to do other cool things with it. And knowing the actually underlying "argument" lets them do more cool things.

In the end its a better annotation, its not a must.

Reply:
@bgruening This is changed in f405106